discourse-doctor.rb

```diff
@@ -116,7 +124,16 @@ def check_plugins
   end
 end
 
+def check_available_disk_space
+  stat = Filesystem.stat(DISCOURSE_PATH.to_s)
+  bytes_available = stat.blocks_available * stat.block_size
+  if bytes_available < 10 * 1024
+    warning("The available disk space at #{DISCOURSE_PATH} is less than 10MB")
```

Comment:
Ok I would keep the 5000 limit and add:

"Run \`apt-get autoremove && apt-get autoclean\` to clean up unused
packages and \`./launcher cleanup\` to remove stale Docker containers."

Still very valid advices.

What would stop us to run automatically "apt-get autoremove && apt-get autoclean" @ZogStriP ?